pages/_app.tsx

```diff
@@ -55,7 +55,7 @@ const MyApp = ({ Component, pageProps }: AppProps): React.ReactElement => {
         <AnalyticsProvider>
           <SWRConfig
             value={{
-              refreshInterval: 5000,
+              refreshInterval: 30000,
```

Comment:
This is a global update to all SWR calls. Might want to consider if there are any calls we want to keep at 5000 before we make this global update.